pom.xml

```diff
@@ -579,5 +579,11 @@
             <version>${asm.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>pl.pragmatists</groupId>
+            <artifactId>JUnitParams</artifactId>
```

Comment:
This project seems to be nearly dead. Are you really sure you want to introduce another dependency for just a single test?